tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -277,6 +277,13 @@ impl<C, N, E, W> Subscriber<C, N, E, W> {
     /// Sets whether or not the formatter emits ANSI terminal escape codes
     /// for colors and other text formatting.
     ///
+    /// When the "ansi" crate feature flag is enabled, ANSI colors are enabled
+    /// by default unless the `NO_COLOR` environment variable is set to
+    /// a non-empty value.  If the `NO_COLOR` environment variable is set to
+    /// any non-empty value, then ANSI colors will be suppressed by default.
+    /// The [`with_ansi`] and [`set_ansi`] methods can be used to forcibly
```

Comment:
@dmlary the docs build on CI is failing because these links are unresolved: https://app.netlify.com/sites/tracing-rs/deploys/64dd3dc4819dce0007d1e8e5

Reply:
Ah, thank you!  Fixed & pushing.